src/mon/MgrMap.h

```diff
@@ -143,6 +143,11 @@ class MgrMap
   // Modules which are enabled
   std::set<std::string> modules;
 
+  // Modules which should always be enabled. A manager daemon will enable
+  // modules from the union of this set and the `modules` set above.
+  unsigned int always_on_version = 0;
```

Comment:
int32_t (must be explicitly sized since we're encoding it)

Reply:
will do. does ::encode not cast to a compatible fixed-size numeric type?

Reply:
resolving to an specific encode method will be the compiler's call, so ends up being platform specific (receiving side's compiler might have a different opinion)

Reply:
ah, right. that seems error prone. i guess other serialization solutions typically have a shema defining the encoded type which would be equivalent having something like encode_{s,u}{32,64} or some such thing.